### PR TITLE
Introduce resolvers for api event

### PR DESCRIPTION
**Pull request by alextargov**
**Description**

Changes proposed in this pull request:
- ...

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/connectivity-adapter/go.sum [hunk 1]
```diff
@@ -120,6 +120,8 @@ github.com/kyma-incubator/compass/components/connector v0.0.0-20230607082115-977
 github.com/kyma-incubator/compass/components/connector v0.0.0-20230607082115-977757624f7e/go.mod h1:K740+tPkGSt9EDSesTwrtHPfRHAiFnj/wJyYaLvuDYU=
 github.com/kyma-incubator/compass/components/director v0.0.0-20230622124112-c6a912d9abe2 h1:6O9u02GejYrbAN5paZWP4hKpENfMG3l2iTDdbVzQE2A=
 github.com/kyma-incubator/compass/components/director v0.0.0-20230622124112-c6a912d9abe2/go.mod h1:F0ddNAyJvaLwGGzR/jo7KMzVUWAn7dhzYAGfvTs2zXs=
+github.com/kyma-incubator/compass/components/director v0.0.0-20230626120331-7d3f35c39b0d h1:1ni3udpgzH4eZcNXUhQPxkO0zA3frlKS4IHo+o3A1sg=
```
Comment (dzahariev) [inline components/connectivity-adapter/go.sum:123]: go mod tidy

**Review by dzahariev — CHANGES_REQUESTED** [review]
Looks good.
Run `go mod tidy` to all components to cleanup the go sum files from old dependencies before we merge this.

**Review by dzahariev — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**